### ISSUE TITLE
Remap string constants containing SRG names (maybe MCP names too)

### ISSUE THREAD

**Issue by coderbot16** [PatchworkMC/patchwork-patcher#5]
AAAAAAAAAAAAAAAAAAAHHHHHHHHHHHHHHHHHH

```java
                if (field.toString().contains("uses") || field.toString().contains("field_222226_d")) {
                    VillagerEvent.usesField = field;
                }
                if (field.toString().contains("maxUses") || field.toString().contains("field_222227_e")) {
                    VillagerEvent.maxUsesField = field;
                }
```

please just use `ObfuscationReflectionHelper`

**Comment by coderbot16** [comment]
brought to you by https://www.curseforge.com/minecraft/mc-mods/infinite-trading

**Comment by coderbot16** [comment]
`TinyRemapper#getRemapper()` will get you most of the way there

**Comment by TheGlitch76** [comment]
Fixed in #20
